WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.swift

```diff
@@ -33,4 +34,31 @@ import Foundation
             return NSLocalizedString("Could not disable notifications", comment: "The app failed to disable notifications for the subscription")
         }
     }
+
+    // MARK: New Comment Threads
+
+    func configuredHeaderView(for tableView: UITableView) -> UIView {
+        guard let post = post else {
+            return .init()
+        }
+
+        let cell = CommentHeaderTableViewCell()
+        cell.backgroundColor = .systemBackground
+        cell.configure(for: .thread, subtitle: post.titleForDisplay(), showsDisclosureIndicator: false)
+
+        // the table view does not render separators for the section header views, so we need to create one.
+        let separatorView = UIView()
```

Comment:
Instead of adding a separator view here, I wonder if we can take advantage of the `UIView` extension that does the same thing?

`cell.contentView.addBottomBorder(withColor: .separator, leadingMargin: tableView.separatorInset.left)`

I only checked one device, but it looks the same to me.

Reply:
TIL! Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/17487/commits/ccf2e6151b6d25e0686aabdeda53f972156da19a. Thank you!